pandas/io/pytables.py

```diff
@@ -1123,7 +1142,7 @@ def append_to_multiple(
 
         # figure out how to split the value
         remain_key = None
-        remain_values = []
+        remain_values: List = []
```

Comment:
Any chance of adding a subtype here?

Reply:
same here and below: i'm pretty sure we'll be able to be more specific in these after #29692